### PR TITLE
add azerty_fr keymap for kprepublic/bm60hsrgb_iso/rev1 

### PR DESCRIPTION
**Pull request by tbourgain**
Add a keymap for french AZERTY layout on KPRepublic bm60hsrgb_iso/rev1 keyboard
Update readme file for rev1 to match keyboard rev (rev2 -> rev1)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kprepublic/bm60hsrgb_iso/rev1/keymaps/azerty_fr/readme.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# French azerty keymap for bm60rgb_iso
+with dynamic macros enabled.
```
Comment (drashna) [inline keyboards/kprepublic/bm60hsrgb_iso/rev1/keymaps/azerty_fr/readme.md:2]:
```suggestion
```

File: keyboards/kprepublic/bm60hsrgb_iso/rev1/keymaps/azerty_fr/rules.mk
Comment (drashna) [inline keyboards/kprepublic/bm60hsrgb_iso/rev1/keymaps/azerty_fr/rules.mk:1]: this file can be removed.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/kprepublic/bm60hsrgb_iso/rev1/keymaps/azerty_fr/readme.md (hunk 1 above)
Comment (tbourgain) [reply keyboards/kprepublic/bm60hsrgb_iso/rev1/keymaps/azerty_fr/readme.md]: file removed

File: keyboards/kprepublic/bm60hsrgb_iso/rev1/keymaps/azerty_fr/rules.mk
Comment (tbourgain) [reply keyboards/kprepublic/bm60hsrgb_iso/rev1/keymaps/azerty_fr/rules.mk:1]: file removed

**Pull request closed without merging**